app/models/supplejack_api/request_metric.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module SupplejackApi
+  # app/models/supplejack_api/request_metric.rb
+  class RequestMetric
+    include Mongoid::Document
+
+    field :date,    type: Date,   default: Time.zone.today
```

Comment:
Make sure that Time.zone.today is the correct method for getting a robust date. IE this could change once it goes to AWS etc.

Reply:
Oh yup, I will do that.